src/codeu/chat/client/core/Controller.java

```diff
@@ -28,9 +28,12 @@
 import codeu.chat.util.Uuid;
```

Comment:
All the below changes should be on the server controller, not client.

https://github.com/lianasyrkett/CodeU-Summer-2017/blob/master/src/codeu/chat/server/Controller.java